<ui version="4.0" >
 <class>FormAgregarReciboBase</class>
 <widget class="QWidget" name="FormAgregarReciboBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:18pt; font-weight:600; color:#000000;">Nuevo Recibo&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dEFechaPago" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Cliente:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CBClientes" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Forma de Pago</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="RBCuentaCorriente" >
          <property name="text" >
           <string>Cuenta Corriente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBContado" >
          <property name="text" >
           <string>Contado</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTexto" >
      <attribute name="title" >
       <string>Recibo de Texto</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Texto</string>
         </property>
         <layout class="QGridLayout" />
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Precio</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_4" >
                  <property name="text" >
                   <string>Importe:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dSBImporte" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="prefix" >
                   <string>$ </string>
                  </property>
                  <property name="maximum" >
                   <double>99999.990000000005239</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="cBRecargos" >
                <property name="text" >
                 <string>Aplicar recargos por pago fuera de termino</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QCheckBox" name="cBPagoMes" >
                <property name="text" >
                 <string>Pago de Mes</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="lmes" >
                  <property name="text" >
                   <string>Mes:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="CBMeses" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="text" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Total:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LETotal" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProductos" >
      <attribute name="title" >
       <string>Recibo de Productos</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
